resources/js/app.js

```diff
@@ -0,0 +1,3 @@
+require('./bootstrap');
+
+require('alpinejs');
```

Comment:
Neither of these are being used and can be removed from `package.json` and from `webpack.mix.js`

Reply:
4c97e63